wiki/en/en-Client-Troubleshooting.md

```diff
@@ -12,6 +12,18 @@ Start with the simple stuff: make sure your instrument/microphone and headphones
 
 **Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4LL setup section](Installation-for-Windows#setting-up-asio4all)
 
+### Getting a loud squeal if you connect to a server
+
+You're creating feedback since your microphone is recording the signal of your speaker. Disconnect from the server and select the correct input and output devices. See the installation page for your OS to get more information on how to do that.
+
+### Not seing the headphone or microphone you just connected to your PC?
+
+Jamulus currently doesn't show devices which were plugged in after the sound system was loaded. Just restart Jamulus with your device plugged in.
+
+### Your sound is stuttering
+
+Probably your audio device doesn't work with the buffer size you selected. Select higher buffer sizes in Jamulus's settings until you sound well. Use another device if possible since high buffer sizes mean high latency. 
```

Comment:
```suggestion
Your audio device may not work with the buffer size you selected. Select a bigger buffer size in Jamulus's settings. If possible though, use another device, since large buffer sizes mean higher latency. 
```